app/Repositories/Category/CategoryRepositoryInterface.php

```diff
@@ -1,7 +1,14 @@
 <?php
 namespace App\Repositories\Category;
 
-interface CategoryRepositoryInterface
+use App\Repositories\RepositoryInterface;
+
+interface CategoryRepositoryInterface extends RepositoryInterface
 {
+    public function indexCate();
```

Comment:
```suggestion
    public function getCategory();
```
viết tên function có ý nghĩa